server/app.py

```diff
@@ -1,21 +1,32 @@
 from flask import Flask, jsonify, request
 
+from qr import AllQr
+
+app = Flask(__name__)
+app.config['CORS-HEADERS']: 'Content-Type'
```

Comment:
Can we remove this and switch to axis use? Should fix cors issue. Or is a new branch planned to fix this?

Reply:
Will be new issue/your event task